lib/jsdom/living/nodes/HTMLElement-impl.js

```diff
@@ -18,25 +18,8 @@ class HTMLElementImpl extends ElementImpl {
     this._clickInProgress = false;
   }
 
-  // Add default event behavior (click link to navigate, click button to submit
-  // form, etc). We start by wrapping dispatchEvent so we can forward events to
-  // the element's default functions (only events that did not incur
-  // preventDefault).
   dispatchEvent(event) {
```

Comment:
Can we remove this method now that it purely forwards?

Reply:
Yes, thanks for the suggestion.